doc/articles/guides/solution-building-single-targetframework.md

```diff
@@ -0,0 +1,83 @@
+---
+uid: Build.Solution.TargetFramework-override
+---
+# Adjust an Uno Solution for a faster build with Visual Studio 2022
+
+The Uno Platform template provides a cross-targeted Class library which includes multiple target frameworks. While building with the command line `dotnet build -f net7.0-ios` does only build the application's head and the class library for `net7.0-ios`, Visual Studio builds all the target frameworks, [regardless of the project head's target framework](https://developercommunity.visualstudio.com/t/Building-a-cross-targeted-project-with-m/651372).
```

Comment:
This wasn't an issue with the shared project approach right?

Reply:
It wasn't indeed. At this point, there are advantages and disadvantages on both techniques, but the class library gives more advantages at this point.